src/com/xilinx/rapidwright/edif/EDIFCellInst.java

```diff
@@ -151,8 +161,20 @@ public EDIFPort getPort(String name){
         return getCellType().getPort(name);
     }
     
+    /**
+     * Gets the sorted ArrayList of EDIFPortInsts on this cell instance as a collection.
+     * @return The collection of EDIFPortInsts on this cell.
+     */
     public Collection<EDIFPortInst> getPortInsts(){
-        return getPortInstMap().values();
+        return portInsts == null ? Collections.emptyList() : portInsts;
+    }
+    
+    /**
+     * Gets the sorted ArrayList of EDIFPortInsts on this cell instance.
+     * @return The sorted list of EDIFPortInst objects on this cell.
+     */
+    public EDIFPortInstList getPortInstList() {
```

Comment:
Does this new method need to exist, since it appears to be identical to `getPortInsts()` above except it returns a specific `EDIFPortInstList` type rather than a `Collection<EDIFPortInst>`? The only new user I'm seeing is:
https://github.com/Xilinx/RapidWright/blob/cd2afa438ca611ba52e1c62a704fa8b5a26a273e/src/com/xilinx/rapidwright/edif/EDIFCellInst.java#L102
which iterates over the object adding each value into a `HashMap`?